sites/public/src/components/listings/ListingsCombined.tsx

```diff
@@ -11,13 +11,10 @@ type ListingsCombinedProps = {
   onPageChange: (page: number) => void
   googleMapsApiKey: string
 }
-
 const ListingsCombined = (props: ListingsCombinedProps) => {
   return (
     <div className={styles["listings-combined"]}>
-      <div className={styles["listings-map"]}>
-        <ListingsMap listings={props.listings} googleMapsApiKey={props.googleMapsApiKey} />
-      </div>
+      <ListingsMap listings={props.listings} googleMapsApiKey={props.googleMapsApiKey} />
```

Comment:
Is there a reason why we remove the div here? Seems like it should stay here to be consistent with the others...and remove it from ListingsMap.tsx?

Reply:
I think the div was just to style it, and now the styling happens in ListingsMap.tsx

Reply:
The div was moved to the component; the component's styling is now within it's definition.